src/schema.ts

```diff
@@ -202,6 +214,11 @@ export default abstract class Schema<
     this.withMutation((s) => {
       s.nonNullable();
     });
+
+    this['~standard'] = createStandardSchemaProps<
```

Comment:
this won't work correctly, b/c it closes over the `schema` instance. Most method calls in yup create new schema instances via cloning and the `validate` method won't keep up with them, e.g. `string().required()['~standard'].validate` will be the nonRequired schema instance

Reply:
You are right, I re-implemented them as getters now which should carry over well with cloning. I added a couple of tests to test that as well.